src/test/resources/edu/cornell/kfs/rass/batch/cu-spring-rass-service-test.xml

```diff
@@ -71,6 +71,9 @@
                 <idref bean="rassProposalProjectDirectorConverter"/>
                 <idref bean="rassAwardProjectDirectorConverter"/>
                 <idref bean="rassPurposeCodeConverter"/>
+                <idref bean="rassAgencyTypeConverter"/>
+                <idref bean="rassProposalStatusConverter"/>
+                <idref bean="rassGrantDescriptionConverter"/>
```

Comment:
Do the unit tests also need some dummy parameter values put in place for the referenced parameters, to help verify that the new classes are mapping the values appropriately? Or does that involve making too many changes to the mock parameter service?

Reply:
Currently the mock parameter service just returns the sub parameter value passed in.

It looks like it would take a fair amount of work to update RassServiceImplTest to verify that the new mappings classes work.  I can make a unit test for one of the mappings class to verify it works atomically.  Do you think that would be sufficient?

Reply:
Yes, having a separate test like that would be good, to help make sure that we're at least unit testing the functionality somewhere.